src/pages/messages/components/MessageList.tsx

```diff
@@ -47,22 +59,26 @@ const MessageItem = ({ message }: MessageItemProps) => {
 
   return (
     <li>
-      <Typography sx={{ display: 'flex', gap: '0.25rem' }}>
-        <span>
-          {senderQuery.isLoading ? (
-            <Skeleton sx={{ width: '8rem' }} />
-          ) : (
-            <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
-              {senderName ? senderName : <i>{t('common.unknown')}</i>}
-            </b>
-          )}
-        </span>
-        <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>
-          ({new Date(message.createdDate).toLocaleString()})
-        </span>
-      </Typography>
+      <Box sx={{ bgcolor: ticketColor[ticketType], p: '0.5rem', borderRadius: '4px', maxWidth: '20rem' }}>
+        <Typography sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <span>
+            {senderQuery.isLoading ? (
+              <Skeleton sx={{ width: '8rem' }} />
+            ) : (
+              <b data-testid={dataTestId.registrationLandingPage.tasksPanel.messageSender}>
+                {senderName ? senderName : <i>{t('common.unknown')}</i>}
+              </b>
+            )}
+          </span>
+          <span data-testid={dataTestId.registrationLandingPage.tasksPanel.messageTimestamp}>
+            ({new Date(message.createdDate).toLocaleString()})
+          </span>
+        </Typography>
+
+        <Divider sx={{ mb: '0.5rem' }} />
```

Comment:
Usikker på farge på denne. Vanskelig å finne noe som passer til alle.

Reply:
I figma brukes dette:
```
Bakgrunn/Bakgrunn 1
#B4A98E
```

Reply:
Tenkte kanskje den ble litt usynlig på kuratorstøtte, så var usikker på om jeg skulle legge inn en annen farge for den🤔

Reply:
kan ta det opp med Cecilie da eventuelt, men ser ut som at det er det som er ønsket, men enig i at det varier litt hvor godt det passer